applications/RomApplication/python_scripts/rom_solver.py

```diff
@@ -35,10 +35,11 @@ def _CreateBuilderAndSolver(self):
             linear_solver = self._GetLinearSolver()
             rom_parameters, solving_strategy = self._ValidateAndReturnRomParameters()
             available_solving_strategies = {
-                "elemental_galerkin": KratosROM.ROMBuilderAndSolver, 
-                "global_galerkin": KratosROM.GlobalROMBuilderAndSolver, 
+                "elemental_galerkin": KratosROM.ROMBuilderAndSolver,
+                "global_galerkin": KratosROM.GlobalROMBuilderAndSolver,
                 "lspg": KratosROM.LeastSquaresPetrovGalerkinROMBuilderAndSolver,
-                "petrov_galerkin": KratosROM.PetrovGalerkinROMBuilderAndSolver
+                "elemental_petrov_galerkin": KratosROM.PetrovGalerkinROMBuilderAndSolver,
+                "global_petrov_galerkin": KratosROM.GlobalPetrovGalerkinROMBuilderAndSolver
```

Comment:
now that we have added RomB&S, GlobalRomB&S, PGRomB&S and GlobalPGRomB&S, we should change the name of LSPGRomB&S to GlobalRomB&S. Right now only the Global version is available.

Reply:
You mean:
LeastSquaresPetrovGalerkinROMBuilderAndSolver -> GlobalLeastSquaresPetrovGalerkinROMBuilderAndSolver?

Reply:
I will change this in another PR, since it is not related direclty to this, one, but I am adding a warning for now...